test/system/static.test.ts

```diff
@@ -633,3 +633,71 @@ test("static scanning NGINX with dockerfile analysis matches expected values", a
     "found gnupg1 in dockerfile packages",
   );
 });
+
+test("inspect registry.suse.com/suse/sle15:15.1", (t) => {
+  const imgName = "registry.suse.com/suse/sle15";
+  const imgTag = "15.1";
+  const img = imgName + ":" + imgTag;
+
+  return plugin.inspect(img, undefined, { experimental: true }).then((res) => {
+    const plugin = res.plugin;
+    const pkg = res.scannedProjects[0];
+
+    t.equal(plugin.name, "snyk-docker-plugin", "name");
+    t.equal(plugin.packageManager, "rpm", "returns rpm package manager");
+
+    t.match(
+      pkg.depTree,
+      {
+        name: "docker-image|" + imgName,
+        version: imgTag,
+        packageFormatVersion: "rpm:0.0.1",
+        targetOS: {
+          name: "sles",
+          version: "15.1",
+          prettyName: "SUSE Linux Enterprise Server 15 SP1",
+        },
+        docker: {},
+      },
+      "root pkg",
```

Comment:
```suggestion
      "dependency tree contains expected data",
```

Just "root pkg" doesn't mean anything much to me...